api/rosetta/construction.go

```diff
@@ -41,19 +41,19 @@ type Construction struct {
 	// their validity or expiration can be determined.
 	retriever Retriever
 
-	// Client is used to submit the constructed transaction to the Flow network.
-	client Client
+	// Submitter is used to submit the constructed transaction to the Flow network.
+	submitter Submitter
```

Comment:
OK, so to clean up the naming a little bit more, I like the pattern where the type is called `Submitter` (noun) and the variable is called `submit` (verb), and then you can call the function something that makes sense in this context, like `Transaction`.

This will then read as `c.submit.Transaction(tx)`, which I think is quite nice. Could also be `Sender` and `send`, then it would read `c.send.Transaction(tx)`. Much better imo than `c.submitter.SendTransaction(tx)`.